pages/common/radare2.md

```diff
@@ -1,7 +1,7 @@
 # radare2
 
 > A set of reverse engineering tools.
-> More information: <https://www.radare.org/r/docs.html>.
+> More information: <https://book.rada.re>.
```

Comment:
```suggestion
> More information: <https://book.rada.re/first_steps/commandline_flags.html>.
```
Lets get the users directly to the information they need.

Reply:
done!